src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_PerfInitPage.js

```diff
@@ -313,15 +320,15 @@ class CJ4_FMC_PerfInitPage {
             { CJ4_FMC_PerfInitPage.ShowPage13(fmc); };
         }
         fmc.onPrevPage = () => {
-            if (fmc.flightPlanManager.getApproachRunway() && fmc.landingQnh > 28 && fmc.landingQnh < 32 && fmc.landingOat && fmc.landingWindDir >= 0 && fmc.landingWindDir <= 360) {
+            if (arrRunwayOutput !== "NO APPROACH RW" && fmc.landingQnh > 28 && fmc.landingQnh < 32 && fmc.landingOat && fmc.landingWindDir >= 0 && fmc.landingWindDir <= 360) {
```

Comment:
i think i liked
`fmc.flightPlanManager.getApproachRunway()` better

Reply:
it would need to be 
if ((fmc.flightPlanManager.getApproachRunway() || fmc.vfrLandingRunway) && fmc.landingQnh > 28 && fmc.landingQnh < 32 && fmc.landingOat && fmc.landingWindDir >= 0 && fmc.landingWindDir <= 360)

I thought that was clumsier, but up to you

Reply:
Hm yeah. Thats is an argument.
In general I would say, always validate against data, not what you have in UI. But this is CJ4 Mod. So move on :D